Gtk-4.0.gir

```diff
@@ -24004,10 +24094,30 @@ To track errors while loading CSS, connect to the
           <type name="CssProvider" c:type="GtkCssProvider*"/>
         </return-value>
       </constructor>
-      <method name="load_from_data" c:identifier="gtk_css_provider_load_from_data">
```

Comment:
Another API change

Reply:
actually that is not an api change, they added a new method and deprecated the old one. The diff is just bad though

Reply:
I see. It also doesn't help that we have to keep this PR growing forever because nobody feels responsible fixing the GLib introspection data even after I made an MR already.

Reply:
Yeah, I think we should merge it already and hopefully glib will get fixed at some point

Reply:
I think at this point we should stop depending on gobject-introspection from the SDK. It just makes no sense. They update GLib but not gobject-introspection, and then all new API has broken annotations.

Reply:
And this should also break other bindings using the SDK. So maybe the SDK should simply not update GLib until there's a corresponding gobject-introspection version. @alatiera

Reply:
Right or we can make the automated script only update other libraries that we can easily fix

Reply:
The problem isn't really updating glib, but that there isn't a g-i update yet :P

Reply:
Yes that's what I'm saying. Both really need to be done in lock-step.